tools/autogen.py

```diff
@@ -289,9 +267,11 @@ def gen_func_implementations(self):
                        'HPy_Dup',
                        'HPy_Close',
                        'HPyArg_Parse',
```

Comment:
kill these two: they are no longer in `public_api.h`, so there is no point to exclude them

Reply:
Fixed!